docs/versioned/getting-started/index.md

```diff
@@ -0,0 +1,95 @@
+# Getting Started with Knative
+
+Welcome! This guide helps you get started with Knative based on your goals and experience level.
+
+## What is Knative?
+
+Knative is a Kubernetes-based platform for deploying and managing serverless workloads. It provides components for building, deploying, and running applications that automatically scale, including scaling to zero when idle.
+
+## Choose Your Path
+
+### I want to try Knative quickly
+
+**Perfect for:** Developers who want hands-on experience
+
+Follow our step-by-step tutorial:
+
+1. [Install Knative locally](quickstart-install.md) - Set up using kind or minikube (10 minutes)
+2. [Deploy your first service](first-service.md) - Create and test a simple application
+3. [Explore autoscaling](first-autoscale.md) - See how Knative scales your app
+4. [Clean up](clean-up.md) - Remove the local installation
+
+**Prerequisites:** Docker, kubectl, 3 CPUs, 3 GB RAM
+
+---
+
+### I want to understand Knative first
+
+**Perfect for:** Evaluators and decision makers
+
+- [What is Knative?](which-knative.md) - Overview of components and use cases
+- [About Knative Functions](about-knative-functions.md) - Simplified function development
+- [Documentation overview](../README.md) - Navigate all Knative docs
+
+Then proceed to the [installation options](../install/) for production deployments.
```

Comment:
I'm not sure that the purchase decision makers are necessarily the same ones who will install Knative in a production environment -- I'd avoid a "then proceed" here.